payments/docker-compose.yml

```diff
@@ -0,0 +1,44 @@
+
```

Comment:
Should just remove unused code right? we already have the skeleton already so don;t have to keep the comment here